src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,72 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int MAX_VALUE = 10;
+    private static final int FIGURE_AMOUNT = 5;
+
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        int randomIndex = new Random().nextInt(FIGURE_AMOUNT);
```

Comment:
Please extract Random as class variable in order not to create new object each time methods are called (same as done for ColorSupplier)